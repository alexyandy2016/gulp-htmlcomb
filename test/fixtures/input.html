<input required="     " class="input     email" value='example@mail.com' type="email" id=inputEmail name="email" data-options="{
    'separator': '@',
    'suffixes': [
      'gmail.com',
      'hotmail.com'
    ]
  }" readonly=     >
