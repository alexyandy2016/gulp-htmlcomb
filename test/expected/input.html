<input class="input email" id="inputEmail" name="email" data-options="{ 'separator': '@', 'suffixes': [ 'gmail.com', 'hotmail.com' ] }" type="email" value="example@mail.com" readonly required>
